build.gradle.kts

```diff
@@ -91,7 +84,9 @@ application {
 }
 
 detekt {
-    failFast = true // fail build on any finding
+    // For now only consider main source files, but ignore test sources
+    input = files("src/main/kotlin")
+    allRules = true // enable all rules, including unstable ones
```

Comment:
`failFast` has been deprecated, see https://github.com/detekt/detekt/pull/3431. When using it, Detekt recommends to use `allRules = true`, since that was apparently the previous behavior of it as well.